src/zenml/model/utils.py

```diff
@@ -64,12 +64,17 @@ def link_step_artifacts_to_model(
         # Implicit linking
         if artifact_config is None and model is not None:
             artifact_config = ArtifactConfig(name=artifact_name)
-            logger.info(
+            logger.debug(
```

Comment:
Can we get rid of this log alltogether? We set a value for artifact config here anyway, which means if we do any linking to a model, we will already enter the `if artifact_config` condition below which contains the other log?

Reply:
Thanks, changed